.env.template

```diff
@@ -4,3 +4,5 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+TRELLO_KEY=your_key
+TRELLO_TOKEN = your_token
```

Comment:
Just a quick note that values in the .env file should actually be strictly of the form `TRELLO_KEY=your_key`
No spaces and no quotes. Flask copes with spaces/quotes but Docker will not. (module 5)